eeduro:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 1000

  arm1_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm1_joint0
    pid: {p: 0.1, i: 0.0001, d: 0.01}
  arm2_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm2_joint0
    pid: {p: 0.1, i: 0.0001, d: 0.01}
  arm3_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm3_joint0
    pid: {p: 0.1, i: 0.0001, d: 0.01}
